### PR TITLE
EOS-22173: Change timeout of blocking call to reduce cpu utilization

### PR DESCRIPTION
**Pull request by selvakumaar5496**
# Problem Statement
- CPU utilization is high when consumer polls infinitely

# Design
- https://jts.seagate.com/browse/EOS-22173

# Coding 
-  Coding conventions are followed and code is consistent [Y/N]: Y
-  Confirm All CODACY errors are resolved [Y/N]: Y

# Testing 
- [ ] Confirm that Test Cases are added (for both the cases, fix and feature) [Y/N]: N (Test cases already exists)
- [ ] Confirm Test Cases cover Happy Path, Non-Happy Path and Scalability [Y/N]: N
- [ ] Confirm Testing was performed with installed RPM [Y/N]:  Y

# Review Checklist 
  Before posting the PR please ensure
- [x] PR is self reviewed
- [ ] Is there a change in filename/package/module or signature [Y/N]: 
- [ ] If yes for above point, Is a notification sent to all other cortx components [Y/N]
- [x] Jira is updated
- [x] Check if the description is clear and explained. 
- [x] Check Acceptance Criterion is defined. 
- [ ] All the tests performed should be mentioned before Resolving a JIRA. 
- [ ] Verification needs to be done before marked as Closed/Verified 

# Documentation
- [ ] Changes done to WIKI / Confluence page

### CODE REVIEW

**Review by sachinpunadikar — APPROVED** [review]
+1

File: py-utils/src/utils/errors.py [hunk 1]
```diff
@@ -123,4 +123,4 @@ class StorageNotFoundError(DataAccessError):
 
 class AmqpConnectionError(Exception):
 
-    """Amqp connection problems"""
+    """Amqp connection problems"""
```
Comment (suryakumar1024) [inline py-utils/src/utils/errors.py:126]: 1) add space arround comments.
2) Add new line at end of script

Comment (selvakumaar5496) [reply py-utils/src/utils/errors.py:126]: added spaces,
We dont add new line at the end of script as per our standard

**Review by rahul-tripathi-git — APPROVED** [review]
+1

**Review by suryakumar1024 — APPROVED** [review]

**Pull request merged**